data_visualization_in_seaborn/data_visualization_in_seaborn.md

```diff
@@ -189,27 +190,17 @@ For more background on scatterplots, watch [this Khan Academy series](https://ww
 
 
 ### Basic scatterplot
-@sage
 
 The seaborn library includes a powerful function called `relplot`, short for "relationships plot" because it's designed to highlight relationships among variables in the data.
 
 If you provide `relplot` with two continuous variables, it will default to making a scatterplot. Here we'll plot age and height.
 
-<div class="python">
-<lia-keep>
-<script type="text/x-sage">
-import pandas as pd
-import numpy as np
-import matplotlib.pyplot as plt
-import seaborn as sns
-covid_data = pd.read_csv("https://raw.githubusercontent.com/kendavidn/yaounde_serocovpop_shared/v1.0.0/data/yaounde_covid_seroprev_dataset.csv")
-
+```python
 sns.relplot(data = covid_data,
             x="val_age", y="val_height_cm")
 plt.show()
-</script>
-</lia-keep>
-</div>
+```
+@Pyodide.eval
```

Comment:
Running this is the first time a learner will have seen that collection of buttons below a plot (home, zoom, pdf, svg, etc.). Would it make sense to have a little explainer box about those buttons? Or are they self-evident enough to not need a tutorial?

Reply:
I retract this, I think it is sufficiently self evident, and if a learner gets confused they can send us a feedback form asking for more info.